credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/constants.py

```diff
@@ -0,0 +1,25 @@
+"""
+Constants for Edly Credentials API.
+"""
+from django.utils.translation import ugettext as _
+
+ERROR_MESSAGES = {
+    'SITE_THEME_DIRECTORY_MISSING': _('Site theme directory not provided.'),
+    'SITE_THEME_UPDATE_SUCCESS': _('Site theme update completed.'),
```

Comment:
where are these constants used?

Reply:
We don't need them for now but all other services have the same ERROR_MESSAGES so I kept them here in case of use in the future.

Reply:
@iam-mhaseeb kindly remove all unnecessary constants and we will add them in case we need to use them in the future.

Reply:
Updated.